test/helpers/cmd.go

```diff
@@ -85,7 +85,7 @@ func (b *CmdStreamBuffer) Filter(filter string) (*FilterBuffer, error) {
 
 	err := json.Unmarshal(b.Bytes(), &data)
 	if err != nil {
-		return nil, fmt.Errorf("could not parse JSON from command %q", b.Cmd())
+		return nil, fmt.Errorf("could not parse JSON from command %q\n%s", b.Cmd(), b.Bytes())
```

Comment:
do you want to see the err itself as well ?

Reply:
Agree, I think printing the error is helpful. The `%s` will convert the bytes to string internally so I think that's fine.

Reply:
Done. Thank to both of you!